website/docs/docs/available-adapters.md

```diff
@@ -5,7 +5,15 @@ id: "available-adapters"
 
 dbt connects to and runs SQL against your database, warehouse, platform, or query engine. It works by using a dedicated **adapter** for each technology. All the adapters listed below are open source and free to use, just like dbt.
 
-Any adapter can be installed from PyPi using `pip`. The installation will include `dbt-core` and any other required dependencies, which may include other adapter plugins. Read more about [installing dbt](dbt-cli/install/overview).
+### Installation
+
+Most adapters can be installed from PyPi using `pip`. The installation will include `dbt-core` and any other required dependencies, which may include other adapter plugins. Read more about [installing dbt](dbt-cli/install/overview).
+
+Some vendor or community adapters may not exist in PyPi. However, an adapter hosted on GitHub can still be installed with `pip`:
```

Comment:
@VersusFacit what do you think about defining the user input? Feel free to enhance my best guesses here:
```suggestion
Some vendor or community adapters may not exist in PyPi. However, you can still install an adapter hosted on GitHub with `pip install`, replacing MAINTAINER_NAME with the person or company maintaining the adapter and ADAPTER_NAME with the adapter you're installing:
```

Reply:
I erred on the side of "let the code say it" as a guess of what to do. More than happy to go with an explicit definition of terms.